analyzer/testdata/src/filtertest/f1.go

```diff
@@ -872,3 +872,25 @@ func detectNode() {
 	nodeTest(rows[0][5], "IndexExpr") // want `true`
 	nodeTest("42", "IndexExpr")
 }
+
+var globalVar string
+var globalVar2 string = time.Now().String() // want `\Qglobal var`
+var globalVar3 = time.Now().String()        // want `\Qglobal var`
+var _ = time.Now().String()                 // false negative // TODO https://github.com/quasilyte/go-ruleguard/issues/369
+var _ string = time.Now().String()          // false negative // TODO https://github.com/quasilyte/go-ruleguard/issues/369
```

Comment:
I think all `_` share the same "object" (none, nil object).
So they define "nothing".
Since you basically should use this `IsGlobal` not for a declaration but rather for the usage of some variable, I don't think these cases matter here.

Reply:
We can't have different types for nil object either. Simply because there is nothing to associate a type with.